hooks/useFeed.tsx

```diff
@@ -8,29 +8,34 @@ const useFeed = ({ feedType }: { feedType: string }) => {
   const { filter } = useFeedProvider();
 
   const fetchPoints = async (start: number) => {
-    const query = supabaseClient.from('posts').select('*').not('likes', 'is', null);
+    if (feedType == FeedType.Recent) {
+      const query = supabaseClient.from('posts').select('*').not('likes', 'is', null);
 
-    if (filter?.platform) {
-      query.eq('platform', filter.platform);
-    }
+      if (filter?.platform) {
+        query.eq('platform', filter.platform);
+      }
 
-    if (filter?.channel) {
-      query.like('parent_url', `%${filter.channel}%`);
-    }
+      if (filter?.channel) {
+        query.like('parent_url', `%${filter.channel}%`);
+      }
 
-    if (feedType == FeedType.Recent) {
       query.order('created_at', { ascending: false });
       query.range(start, start + 5);
-    } else {
-      query.order('likes', { ascending: false });
-      query.range(start, start + 20);
-    }
 
-    const { data: posts } = await query.returns();
+      const { data: posts } = await query.returns();
 
-    return {
-      posts,
-    };
+      return {
+        posts,
+      };
+    } else {
+      const { data: posts } = await supabaseClient
+        .from('trending_posts')
+        .select('*')
+        .order('score', { ascending: false })
+        .range(start, start + 20); // Adjust limit as needed
```

Comment:
remove dev comments 🚮 
```suggestion
        .range(start, start + 20); 
```